amoro-ams/src/test/java/org/apache/amoro/server/optimizing/maintainer/TestDataExpire.java

```diff
@@ -194,17 +194,11 @@ private void testUnKeyedPartitionLevel() {
 
     List<Record> expected;
     if (tableTestHelper().partitionSpec().isPartitioned()) {
-      if (expireByStringDate()) {
-        expected =
-            Lists.newArrayList(
-                createRecord(2, "222", parseMillis("2022-01-03T12:00:00"), "2022-01-03T12:00:00"));
-      } else {
-        expected =
-            Lists.newArrayList(
-                createRecord(2, "222", parseMillis("2022-01-03T12:00:00"), "2022-01-03T12:00:00"),
-                createRecord(3, "333", parseMillis("2022-01-02T12:00:00"), "2022-01-02T12:00:00"),
-                createRecord(4, "444", parseMillis("2022-01-02T19:00:00"), "2022-01-02T19:00:00"));
-      }
+      expected =
```

Comment:
It seems the table was partitioned by `op_time` with `day` transformer, and will expire the data before 86400s, so the record created in `2022-01-02` and `2022-01-03` will still be there(because the partition can't be expired) is this understanding correct?

In the other way, is there any way we can know the partition for each record easier?

Reply:
can we add some comments to describe them here? thanks